tools/infer/utility.py

```diff
@@ -246,7 +246,7 @@ def create_predictor(args, mode, logger):
                     logger.info("Please keep your paddlepaddle-gpu >= 2.3.0!")
 
         elif args.use_npu:
-            config.enable_custom_device("npu")
+            config.enable_npu()
```

Comment:
please explain the reason for making this change. it seems irrelevant to the purpose of the PR.

Reply:
sorry, this change is wrong. Now have re-PR: https://github.com/PaddlePaddle/PaddleOCR/pull/10421